etc/schema.json

```diff
@@ -2079,7 +2079,10 @@
                 "dest_port": {
                     "type": "integer"
                 },
-                "elephant": {
+                "elephant_toclient": {
```

Comment:
Not a big fan of this format, was this already decided ?

Reply:
Changing the type of the `elephant` object would also hinder the backwards compatibility